xdocs/usermanual/component_reference.xml

```diff
@@ -6078,6 +6080,34 @@ This can be useful if your JDBC Sample changes some data and you want to reset s
 In the linked test plan, "<code>JDBC PostProcessor</code>" JDBC PostProcessor calls a stored procedure to delete from Database the Price Cut-Off that was created by PreProcessor.
 </p>
 <figure width="818" height="399" image="jdbc-post-processor.png">JDBC PostProcessor</figure>
+</component>
+
+<component name="JSON PostProcessor" index="&sect-num;.8.9">
+<description>
+<p>
+The JSON PostProcessor enables you extract data from JSON responses using JSON-PATH syntax. This post processor is very similar to Regular expression extractor.
+It must be placed as a child of HTTP Sampler or any other sampler that has responses.
+It will alow you to extract in a very easy way text content, see <a href="http://goessner.net/articles/JsonPath/" >JSON Path syntax</a>.
+
+</p>
+</description>
+<properties>
+    <property name="Name" required="No">Descriptive name for this element that is shown in the tree.</property>
+    <property name="Variable Names" required="Yes">Semi-colon separated names of variables that will contain the results of JSON-PATH expressions (must match number of JSON-PATH expressions)</property>
+    <property name="JSON Path Expressions" required="Yes">Semi-colon separated JSON-PATH expressions (must match number of variables)</property>
+    <property name="Default Values" required="No">Semi-colon separated default values if JSON-PATH expressions do not return any result(must match number of variables)</property>
+    <property name="Match Numbers" required="No">If the JSON Path query leads to many results, you can choose which one(s) to extract as Variables:
+		<ul>
+		<li>0 : means random (Default Value)</li>
```

Comment:
I would place code like fragments in `<code>` tags, use a definition list instead of an unordered list and escape html entities like `<` and `>`. If you want to show user input, you could use `<em>`, e.g. `<code><em>variable name</em>_N</code>`.

Xth could be written as `X<sup>th</sup>` and `> to number` could  be spelled out as `greater than number`